packages/ocamlgraph/ocamlgraph.1.8.8/opam

```diff
@@ -22,14 +22,12 @@ tags: [
   "imperative"
 ]
 build: [
-  ["autoconf"]
```

Comment:
You need to have `["touch" "autoconf"]` here I think, see the opam file for 1.8.7.

Reply:
Fixed. Thanks for noticing.